.ci/travis/xtr_releasenotes-gen.sh

```diff
@@ -61,11 +67,6 @@ echo ==============================================
 cat twitter.txt
 echo ==============================================
 echo
-echo "google plus post:"
-echo ==============================================
-cat gplus.txt
```

Comment:
Why is this being removed?

Reply:
there is no gplus anyone - https://github.com/checkstyle/contribution/pull/434